CMakeLists.txt

```diff
@@ -99,6 +99,8 @@ Cabana_add_dependency( PACKAGE Heffte VERSION 2.0.0 )
 # enable unit tests
 option(Cabana_ENABLE_TESTING "Build tests" OFF)
 if(Cabana_ENABLE_TESTING)
+  set(TEST_HARNESS_DIR ${CMAKE_SOURCE_DIR}/core/test_harness)
```

Comment:
I thought we wanted the test harness to live in `Cabana/cmake/test_harness`?

Reply:
You're right - moved